src/sentry/api/authentication.py

```diff
@@ -232,3 +242,29 @@ def authenticate_credentials(self, request: Request, token):
             scope.set_tag("api_project_key", key.id)
 
         return (AnonymousUser(), key)
+
+
+FORWARDED_SSL_CERT_HEADER_NAME = "ssl-client-cert"
```

Comment:
Will be hammered out in an upcoming notion doc.  But I think this closely matches what I've seen can be done with simple nginx or k8s configs.